Libplanet/Blockchain/Policies/IBlockPolicy.cs

```diff
@@ -18,6 +20,11 @@ public interface IBlockPolicy<T>
         /// </summary>
         IAction BlockAction { get; }
 
+        /// <summary>
+        /// A method that determines if the transaction is valid.
+        /// </summary>
+        Func<Transaction<T>, bool> IsValidTransaction { get; }
```

Comment:
How about changing `IsValidTransaction()` to simple predicate instead of returning `Func`?

```csharp
bool IsValidTransaction(Transaction<T> transaction)
```

Reply:
Changed to a normal method and renamed to `IsTransactionValid`.